tools/hist/fio-histo-log-pctiles.py

```diff
@@ -396,7 +397,7 @@ def compute_percentiles_from_logs():
     print('buckets per group = %d' % buckets_per_group)
     buckets_per_interval = buckets_per_group * args.bucket_groups
     print('buckets per interval = %d ' % buckets_per_interval)
-    bucket_index_range = range(0, buckets_per_interval)
+    bucket_index_range = list(range(0, buckets_per_interval))
```

Comment:
Again I'm little unsure why the iterator would not be good enough..

Reply:
I have removed "list()" again.